cpp/benchmarks/io/orc/orc_reader_input.cpp

```diff
@@ -63,18 +64,22 @@ void BM_orc_read_data(nvbench::state& state,
   auto const d_type                 = get_type_or_group(static_cast<int32_t>(DataType));
   cudf::size_type const cardinality = state.get_int64("cardinality");
   cudf::size_type const run_length  = state.get_int64("run_length");
+  cuio_source_sink_pair source_sink(IOType);
 
-  auto const tbl =
-    create_random_table(cycle_dtypes(d_type, num_cols),
-                        table_size_bytes{data_size},
-                        data_profile_builder().cardinality(cardinality).avg_run_length(run_length));
-  auto const view = tbl->view();
+  auto const num_rows = [&]() {
```

Comment:
Would `num_rows_written` improve this for you?
I'm not sure how to approach this differently.
FWIW, I could return the table from the lambda and then only keep the number of rows.

Reply:
It isn't a major issue, just a stylistic thing. `num_rows_written` is a better name because it would let you know it's not just calculating the number of rows.

Reply:
renamed